src/main/resources/add-ons-to-index.json

```diff
@@ -586,6 +604,9 @@
       "uid": "org.openmrs.module.databasebackup",
       "type": "OMOD",
       "name": "Database Backup",
+	  "tags": [
+        "database"
```

Comment:
"database" is not a helpful tag. It's not a category of addons that's useful to group together.
Maybe administration or sysadmin would be better